<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>First_Window</class>
 <widget class="QMainWindow" name="First_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="squat_reps_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="squat_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="one_rep_calc_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate One Rep Max</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="squat_weight_box">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="deadlift_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="deadlift_weight_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="deadlift_reps_box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="bench_reps_box">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bench_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="bench_weight_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ohp_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="ohp_weight_box">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ohp_reps_box">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="specialValueText">
     <string>Butts</string>
    </property>
   </widget>
   <widget class="QLabel" name="squat_img">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imgs/Back_Squat.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="deadlift_img">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imgs/Deadlift.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bench_img">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imgs/Bench.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ohp_img">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imgs/OHP.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDick">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuDick"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
